utils/schemavalidation/schemavalidation.go

```diff
@@ -0,0 +1,43 @@
+//go:generate packr2
+
+package schemavalidation
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gobuffalo/packr/v2"
+	"github.com/pkg/errors"
+	"github.com/qri-io/jsonschema"
+)
+
+// newSchemaBox returns a *packer.Box with the schema files from the schema sub-directory
+func newSchemaBox() *packr.Box {
+	return packr.New("github.com/cnabio/cnab-go/utils/schemavalidation/schema", "./schema")
+}
+
+// Validate validates the provided objectBytes against the provided CNAB-Spec schemaType
+func Validate(schemaType string, objectBytes []byte) ([]jsonschema.ValError, error) {
```

Comment:
I would rather expose a separate method for bundle, claim, etc. schema validation, as this presents a clearer API to end users.

It would also be nice if we had convenience methods that allow passing the native go types rather than raw bytes, as this would provide type checking at compile time (though admittedly at the cost of re-serializing data twice in some cases).

Reply:
Good call.  Updated to add `ValidateBundle`, `ValidateClaim`.  Follow-ups to add convenience methods sounds great if you'd like to take that on.